### DISCUSSION TITLE
Why revert the std::move on the std::shared_ptr?

### DISCUSSION THREAD

**Discussion by portsip** [resiprocate/resiprocate#314] [General]
Hi, I see the changes on December 28, 2022 that Daniel submitted, all std::move on the shared_ptr are removed, what's reason for this?
It's seems no problem to use the std::move?

@dpocock

**Comment by sgodin** [comment]
There is an ongoing discussion about this happening offline (between @bwalden , @dpocock and myself).  Some of us believe the change was unnecessary, but we are awaiting feedback from @dpocock on what he was seeing to cause this change.  Unfortunately he has been missing on the list for a while now.
